Readme.md

```diff
@@ -20,6 +20,7 @@ Helpful Libraries consists of the following independent libraries (all in their
 - [LINQ to DB Libraries for Orchard Core](Lombiq.HelpfulLibraries.LinqToDb/Readme.md)
 - [Orchard Core Libraries](Lombiq.HelpfulLibraries.OrchardCore/Readme.md)
 - [RestEase Libraries for Orchard Core](Lombiq.HelpfulLibraries.RestEase/Readme.md)
```

Comment:
Add a note to this page that the Refit one is recommended and it's not part of the main HL package anymore.